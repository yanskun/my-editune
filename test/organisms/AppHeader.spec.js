import { createLocalVue, shallowMount } from "@vue/test-utils"
import ElementUI from "element-ui"
import Component from "~/components/organisms/AppHeader"

const localVue = createLocalVue()
localVue.use(ElementUI)

describe("AppHeader", () => {
  test("snapshot", () => {
    const wrapper = shallowMount(Component, {
      propsData: {}, localVue
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
