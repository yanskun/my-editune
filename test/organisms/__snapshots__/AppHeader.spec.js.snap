// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader snapshot 1`] = `
<el-header-stub
  class="header"
  height="60px"
>
  <logo-stub
    class="logo"
    color="white"
  />
</el-header-stub>
`;
